### ISSUE TITLE
regulates o part_of->regulates is in go_plus

### ISSUE THREAD

**Issue by ukemi** [geneontology/go-ontology#13926]
We should not be inferring that if occurent1 regulates occurent2 and occurent2 is part_of occurent3, then occurent1 regulates occurent3. This is the case in go_plus and results in inaccurate inferred annotations generated by GO-CAM models, but these property chains are not in RO.

These also exist for the positive and negative properties.

**Comment by balhoff** [comment]
@cmungall first item flushed out by the explanations view.

**Comment by ukemi** [comment]
@vanaukenk  I think this will also address the issue with the Wnt model.

The model @balhoff and I looked at was this one:
http://noctua.berkeleybop.org/editor/graph/gomodel:586fc17a00001081

**Comment by vanaukenk** [comment]
@balhoff and @ukemi  - thanks.  
@balhoff  - I didn't quite understand your comment above.  Has this property chain been fleshed out, or is it on the chopping block to be fleshed out?  The Wnt model still shows the inferences:
http://noctua.berkeleybop.org/editor/graph/gomodel:5966411600000538

**Comment by balhoff** [comment]
@vanaukenk I just meant that this was the first problem explained by the explanations view. Still needs to be fixed!

**Comment by dosumis** [comment]
Long standing ticket on flushing #12344.  Agree needs to be v. high priority. 

TL;DR:

Chain removed from RO.  
Before removing from GO we should instantiate the existing inferences as most are correct.  If we tag these in some way they could be reviewed and bad ones weeded out.

**Comment by ukemi** [comment]
@dougli1sqrd will query for all of the inferred regulates relationships based on this property chain. The next step will be for an editor to go through them and determine which ones should be asserted. Then we will assert the correct ones and drop this property chain from GO-PLUS.

**Comment by ukemi** [comment]
This project was discussed at the hackathon. We determined that if process/functionA regulated process/functionB and functionB is a part of processC, then process/functionA only regulates processC if process/functionA is causally_upstream_of and not part_of processC.

This means that the property chain regulates_o_part_of is not always valid. @balhoff determined that there are about 1700 relationships that will be lost when we remove this property chain. There are two ways forward: 1) Assert all of the currently inferred regulates relationships in the ontology and then remove the property chain. 2) Manually check the inferred relationships and add them by hand to the ontology. I prefer the latter.

Second decision: Should we create regulates relations between the processes, or create subclass relations to regulation terms? There are pluses and minuses to each approach. The plus of the former is that the relationships can be made directly to the parent of the parthood relationship, but then these would need to be completely maintained by hand. The plus of the latter is that the asserted relations would be subsumption, so once the relationship is made, reasoning will take care of the rest. The minus of the latter would be that not all regulation of x terms are instantiated, so some part_of chains would be skipped.

**Comment by cmungall** [comment]
re second decision: we can start with subclass and migrate to more explicit
relations later

>

**Comment by ukemi** [comment]
Thanks @cmungall. @balhoff can you do that automatically? If you can, we can go through your list and delete ones that we think are incorrect.

**Comment by balhoff** [comment]
Here is the list of inferences from these property chains: https://docs.google.com/spreadsheets/d/16Foto8g-0iy71OXqTnc0DV1ZvTIc0IjX10WmksPghfA/edit#gid=0

**Comment by pgaudet** [comment]
Is this done ?

**Comment by ukemi** [comment]
Nope.

**Comment by balhoff** [comment]
Yep. Fixed by #18653.
